cli/plasmo/src/features/env/env-config.ts

```diff
@@ -16,6 +15,7 @@ type LoadedEnvFiles = Array<{
   contents: string
 }>
 
+const JSON_DIRECTIVE_REGEX = /^\s*~/
```

Comment:
Loving this concept of a custom directive for JSON env - I think we should make it explicit that it is json and that it will be JSON.stringified. Perhaps instead of a `~`, we should regex for a `json(*)`?

Reply:
Yeah, I figured this would be something you choose. But conceptually, yes, I wanted to make it obviously explicit that it would be JSON parsed.

For clarity are you suggesting something like:

```
VAR=`JSON(
{"key": "val"}
)`
```

Reply:
Yup I think that'd be best. If we can make something like this work that'd be pretty cool too tho might be stretch goal:

```
VAR=`json({
  "key": "val"
})`
```

or 

```
VAR=`json({"key": "val"})`
```

Reply:
Fixed in 9ca5babf672545f1c6fbcc96594589b06200e1ba